### ISSUE TITLE
CI is failing

### ISSUE THREAD

**Issue by gnestor** [jupyterlab/jupyter-renderers#211]
I'm seeing some consistent CI build failures but I don't understand what's causing them.

```
RemoveError: 'setuptools' is a dependency of conda and cannot be removed from conda's operating environment.
````
https://travis-ci.org/jupyterlab/jupyter-renderers/builds/563656110#L667

@scopatz

**Comment by dhirschfeld** [comment]
RElated? https://github.com/conda/conda/pull/9014

**Comment by scopatz** [comment]
See fix in #212

**Comment by gnestor** [comment]
@dhirschfeld Yes, totally related!
